.github/workflows/release.yml

```diff
@@ -252,17 +252,23 @@ jobs:
           # use our custom token for more permissions, e.g. "workflow" which is needed to push workflow files
           git config --local "http.https://github.com/.extraheader" "AUTHORIZATION: basic $(echo -n pat:${HRA_GITHUB_TOKEN} | base64)"
 
-      - name: Push the target branch
-        if: ${{ needs.vars.outputs.dry_run == 'false' }}
```

Comment:
instead of the condition on the github syntax there's now a condition in the shell code